src/System.Private.CoreLib/src/Internal/Runtime/CompilerHelpers/MathHelpers.cs

```diff
@@ -41,8 +41,8 @@ private static UInt32 Hi32Bits(UInt64 a)
             return (UInt32)(a >> 32);
         }
 
-        [RuntimeExport("LMulOvf")]
-        public static Int64 LMulOvf(Int64 i, Int64 j)
+        [RuntimeExport("LMulOfv")]
```

Comment:
It should be LMulOvf. This typo is on the other side. 

Could you please undo this change, and replace LMulOfv with LMulOvf?